googletest/src/gtest.cc

```diff
@@ -2979,13 +2979,15 @@ void ColoredPrintf(GTestColor color, const char* fmt, ...) {
   CONSOLE_SCREEN_BUFFER_INFO buffer_info;
   GetConsoleScreenBufferInfo(stdout_handle, &buffer_info);
   const WORD old_color_attrs = buffer_info.wAttributes;
-
+  // Let's reuse the BG
+  const WORD existing_bg = old_color_attrs & 0x00F0;
```

Comment:
I think we should probably define a local mask that includes these fields from the [character attributes documentation](
https://msdn.microsoft.com/en-us/library/windows/desktop/ms682088(v=vs.85).aspx#_win32_character_attributes)

    const WORD background_mask =
      BACKGROUND_BLUE |       // Background color contains blue.
      BACKGROUND_GREEN |      // Background color contains green.
      BACKGROUND_RED |        // Background color contains red.
      BACKGROUND_INTENSITY;   // Background color is intensified.

So we can at least give a name and a single maintenance point to this concept.